yt_dlp/extractor/youtube.py

```diff
@@ -763,6 +793,13 @@ def _extract_video(self, renderer):
             'duration': duration,
             'view_count': view_count,
             'uploader': uploader,
+            'channel_id': channel_id,  # TODO: validate?
+            'timestamp': timestamp,
```

Comment:
Don't set timestamp, only upload date. Since the video extractor doesn't return its own timestamp, this will be propagated